pkg/controller/studyjob/studyjob_controller.go

```diff
@@ -289,7 +289,7 @@ func (r *ReconcileStudyJobController) checkStatus(instance *katibv1alpha1.StudyJ
 						job := &batchv1.Job{}
 						joberr := r.Client.Get(context.TODO(), nname, job)
 						if joberr == nil {
-							if err := r.Delete(context.TODO(), job, client.PropagationPolicy(metav1.DeletePropagationForeground)); err != nil {
```

Comment:
If you remove `DeletePropagationForeground`, the pods generated by Job or CronJob remain after the studyjob was deleted.
You should delete them explicitly after a Job or a CronJob is deleted.